src/components/Footer/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import { styled } from '@mui/system';
+// import { useTheme } from '@emotion/react';
+import theme from '../../styles/Theme';
```

Comment:
I updated this import theme to use your created styling, is this ok @kiranacascade ?

Because there was some lint error in `theme.palette.secondary.main` when I run the test

Reply:
hi kak @fatjan I have fixed the import for the `{useTheme}` based on this documentation:
https://mui.com/material-ui/customization/theming/#theme-builder

everything is working fine. I think you can merge it now.